geoips/plugins/modules/readers/abi_netcdf.py

```diff
@@ -21,6 +21,7 @@
 
 from scipy.ndimage import zoom
 
+from geoips.interfaces import readers
```

Comment:
This does not cause any sort of circular import?   Importing the interface from the plugin?